recipes-devtools/clang/libclc_git.bb

```diff
@@ -14,6 +14,7 @@ inherit cmake pkgconfig python3native qemu
 SPIRV_DEP = "${@'spirv-tools spirv-tools-native' if int(d.getVar('MAJOR_VER')) >= 12 else ''}"
 
 DEPENDS_append = " qemu-native clang ${SPIRV_DEP}"
+DEPENDS_append_class-target = " llvm-common"
```

Comment:
i wonder if it should br in DEPENDS of clang recipe

Reply:
See my question from the first libclc PR: https://github.com/kraj/meta-clang/pull/429#issuecomment-778755847
No other llvm subproject recipe has it.

Reply:
But sure, the clang recipe should likely merge llvm-common and install llvm-common (and the versioned variant) in the crossscripts directory.

Reply:
Alternative: https://github.com/kraj/meta-clang/pull/433